lib/cisco_node_utils/cmd_ref/system.yaml

```diff
@@ -1,8 +1,7 @@
 # system
 ---
 resources:
-  nexus:
-    get_command: "show system resources"
-  # ios_xr:
-    # grpc doesn't allow this command
-    # get_command: 'monitor threads iteration 1'
+  # grpc doesn't allow this command
+  # get_command: 'monitor threads iteration 1'
```

Comment:
No biggie, just wondering. Was this needed to get the 'validate_property_excluded' check to work?